lib/include/openamp/rpmsg_virtio.h

```diff
@@ -53,6 +71,7 @@ struct rpmsg_virtio_shm_pool {
 struct rpmsg_virtio_device {
 	struct rpmsg_device rdev;
 	struct virtio_device *vdev;
+	struct rpmsg_virtio_config config;
```

Comment:
Nitpick: Shouldn't this go just below rdev, putting the two structs before the various pointers? This generally leads to slightly better alignment.